pix2tex/dataset/latex2png.py

```diff
@@ -143,10 +143,10 @@ def tex2png(eq, **kwargs):
     return __cache[eq]
 
 
-def tex2pil(tex, error_index=False, **kwargs):
+def tex2pil(tex, return_error_index=False, **kwargs):
     pngs, error_index = Latex(tex, **kwargs).write(return_bytes=True)
     images = [Image.open(io.BytesIO(d)) for d in pngs]
-    return (images, error_index) if error_index else images
+    return (images, error_index) if return_error_index else images
```

Comment:
I think now its correct @TITC. Before there were 2 errors:
1. the class `Latex` would receive `error_index` as kwarg without handling it
2. tex2pil would always return a tuple. (images, error_index) if return_error_index is true and (images, images) if not

I also introduced an error by naming it the same way as the output variable.

Reply:
> * the class `Latex` would receive `error_index` as kwarg without handling it

what's the problem if kwarg has that parameter without handling it?

Reply:
> 2\. tex2pil would always return a tuple. (images, error_index) if return_error_index is true and (images, images) if not

Wow, break my recognition. It's true！
```python

def foo(b):
    a = b**2
    return a, b if b > 10 else a


if __name__ == '__main__':
    print(foo(2))

```
output
```shell
(4, 4)
```

Reply:
Because it would throw an error in the  `__inti__` function of the `Latex` class: https://github.com/lukas-blecher/LaTeX-OCR/blob/4f7ba9fc5ea171ed470f2a0444c51cc14780a48c/pix2tex/dataset/latex2png.py#L27
There wouldn't be a problem if there was a `**kwargs` there but I also think its more beautiful this way.

Reply:
can't agree anymore, I also prefer this style.